e2e/tests/conceptsets/conceptset-edit.spec.ts

```diff
@@ -26,9 +25,7 @@ describe('Editing and Copying Concept Sets', () => {
    */
   test('Add to existing Concept Set in same workspace', async () => {
     // Create a workspace
-    const workspaceCard = await findWorkspace(page, true);
-    const workspaceName = await workspaceCard.getWorkspaceName();
-    await workspaceCard.clickWorkspaceName();
+    const workspaceName = await findWorkspace(page, true).then(card => card.clickWorkspaceName());
```

Comment:
unrelated change. while running and inspecting test code, I realized 3 lines of code can be rewritten in one line.